src/sentinel.c

```diff
@@ -45,8 +45,8 @@
 extern char **environ;
 
 #if USE_OPENSSL == 1 /* BUILD_YES */
-extern SSL_CTX *redis_tls_ctx;
-extern SSL_CTX *redis_tls_client_ctx;
+extern SSL_CTX *server_tls_ctx;
+extern SSL_CTX *server_tls_client_ctx;
```

Comment:
I think "server client" is extremely confusing.

There is one server certificate for when valkey acts as a server and one client certificate used when valkey acts as a client, i.e. when it connects to other nodes.

Where it acts as a client, we can't call it server. 🙃

I can accept tls-server-ctx and tls-client-ctx or something like that.

Reply:
I never liked the current names either.

Rebranding in sentinel.c and tls.c is problematic.  Please see https://github.com/valkey-io/valkey/issues/269 for my initial thoughts on these two files.  It would be good if we can handle these two global variables in a manner that is consistent with the rebranding for the rest of these two files.

It seems to me that the most consistent approach would be to rebrand these two files using the rule "redis -> sentinel" instead of "redis -> server".

How do you feel about "sentinel_tls_server_ctx" and "sentinel_tls_client_ctx"?

Reply:
Hm... isn't it `external`? I think it's used by valkey-server too.

Reply:
Yeah, these are declared in src/tls.c and they are used by valkey-server too, including the cluster bus (if i'm not completely lost).

Reply:
That's right, `extern` in sentinel.c.
~~~
extern SSL_CTX *redis_tls_ctx;
extern SSL_CTX *redis_tls_client_ctx;
~~~

Non-static and assigned (therefor allocated and globally visible) in tls.c:
~~~
SSL_CTX *redis_tls_ctx = NULL;
SSL_CTX *redis_tls_client_ctx = NULL;
~~~

I checked, both by searching and compiling with variations.  Those two symbols are only referenced in these two files, and only when the build-flags are right.  Nearly all of tls.c is static.  The only functions that are referenced by the server code are RedisRegisterConnectionTypeTLS, ValkeyModule_OnLoad, and ValkeyModule_OnUnload.

If minimizing risk is the goal, then I would use the rule "redis -> valkey".  It won't introduce any NEW ways to confuse the reader.

Reply:
Yeah, `valkey-server` is using it via tls.c, createTLSConnection().

1. redis -> valkey, ok

       SSL_CTX *valkey_tls_ctx = NULL;
       SSL_CTX *valkey_tls_client_ctx = NULL;

2. No prefix, ok

       SSL_CTX *tls_ctx = NULL;
       SSL_CTX *tls_client_ctx = NULL;

3. No prefix, but add "server" to the server ctx, to use the same naming style as the tls_client_ctx. Is that confusing? (IMHO, I think it makes it more clear. Server means incoming connections; client is doing outgoing connections.) ok

       SSL_CTX *tls_server_ctx = NULL;
       SSL_CTX *tls_client_ctx = NULL;

Reply:
↑↑↑ Chose one of the above, ok?

Reply:
OK, I choose (1), because "redis -> valkey" has the least semantic impact.  If you were used to reading the old code, the new code won't be surprizing.

Reply:
done